### ISSUE TITLE
import 使用重命名，就挂了

### ISSUE THREAD

**Issue by fiveLucky** [ant-design/babel-plugin-import#256]
import { SomeCom  as Com } from 'antd'

转换后

var Com = require('antd/Com')

而不是
var Com = require('antd/some-com')

**Comment by nvana** [comment]
I'm facing the same issue.

`import { Input as Input2 } from 'antd';`

Ends up with `Error: Cannot find module 'antd/lib/input2'`

**Comment by zzx18023** [comment]
`plugins: [
    [
      'component',
      {
        libraryName: 'mint-ui',
        style: true
      },
      'mint-ui'
    ],
    [
      'component',
      {
        libraryName: 'element-ui',
        styleLibraryName: 'theme-chalk'
      },
      'element-ui'
    ]
  ]`



`import { Button as mtButton } from 'mint-ui'`
`import { Button as elButton } from 'element-ui'`

`Error: Cannot find module 'element-ui/lib/el-button'`
`Error: Cannot find module 'mint-ui/lib/mt-button'`

**Comment by chengjianhua** [comment]
@sorrycc @nvana I have submitted a pr to fix this problem. Basically, the error occurs because of non-consistent argument pass for the method to generate an import declaration.

The interesting part is that if the import will be transformed into the reduced on account on whether the specifier inside source import was been used in any `MemberExpression`, `VariableDeclarator` ... nodes.

In my cases, the behavior of generating import declaration inside `VariableDeclarator` handler isn't consistent with the most correct ones, so replace renamed `local.name` with `imported.name` solved this.

I just encountered this on `VariableDeclarator`. So for the sake of saving my time, I just updated the logics of `this.buildVairableDeclarator`, I didn't touch others.

If there're any other relating issues, I'd be happy to dive into it and ensure this is fixed completely. 😃

----

I hope that @sorrycc @yesmeck have time to review this pr ASAP, thx~ 🙏
